packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -11987,63 +11987,65 @@ export class AdapterClass extends EventEmitter {
             for (const instObj of instanceObj.instanceObjects) {
                 const obj: IoPackageInstanceObject & { state?: unknown } = instObj;
 
+                // the object comes from non-checked io-package, so treat the id as unknown
+                if (!obj || typeof (obj._id as unknown) !== 'string' || (!obj._id && obj.type !== 'meta')) {
```

Comment:
Ok, so only allowed on top level with empty _id is meta ... k

Reply:
yes, you can make e.g. `hm-rpc.0` a meta object this way. It was that way before.